frontend/www/js/omegaup/components/course/Form.vue

```diff
@@ -211,7 +211,7 @@
   </div>
 </template>
 
-<style>
+<style lang="scss">
```

Comment:
perdón >< se me olvidó esto!

en general siempre queremos tener `scoped` en el `style` excepto en _muy_ contados casos donde hay manipulación de DOM (como con el Markdown).

```suggestion
<style lang="scss" scoped>
```

Reply:
Aquí me quedó una duda: 

Ese estilo que estoy agregando en este archivo ya lo tiene el componente de `Autocomplete.vue`, pero por alguna razón no se ve reflejado en esta vista(Curso nuevo), pero en la vista de Editar curso si se ve bien, aún cuando los dos utilizan el mismo componente: `Form.vue`. Revisé y ninguno de los componentes mencionados tienen el `scoped` en la etiqueta.

¿A que otra cosa se deberá?

Reply:
ugh, posiblemente es porque como el DOM se manipula en tiempo de ejecución por el plugin de bootstrap y este no se comunica con Vue para nada, a los nodos del DOM que se crean no se les agrega el prefijo correcto (a final de cuentas lo único que hace `scoped` es agregar un [atributo random](https://vue-loader.vuejs.org/guide/scoped-css.html) a todos los elementos y las reglas de CSS).

puedes dejar el `scoped` y mover ese estilo a https://github.com/omegaup/omegaup/blob/master/frontend/www/sass/base/_common.scss ?

Reply:
Listo.